ROOTFS/etc/waggle/sanity/fatal/cameras-present.test

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
@seanshahkarami and @gemblerz please coordinate on this.  this test does a test on the count of cameras detected on the network.  and the change that Yongho is working (https://github.com/waggle-sensor/waggle-sanity-check/pull/2) does something similar.

I think its is appropriate to have multiple camera tests.  1 (like this one) that just detects if the cameras are present on the network and a 2nd (Yongho's) that actually tests if the cameras work.

Just please coordinate.

Reply:
Sure. Yeah, looks like Omar already has something working for detecting if all are present.

Just for the fact you have limited space to encode your results, I'd agree that keeping those checks split seems reasonable.